### PR TITLE
hj350_strassen

### PR DESCRIPTION
**Pull request by hj350**
ok

### CODE REVIEW

**Comment by hj350** [comment]
All if fine

**Comment by wilslow** [comment]
lol

File: hj350-strassen.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Nice town, now living in Mamer
```
Comment (Treverer) [inline hj350-strassen.md]: Why did you leave? @hj350

Comment (hj350) [reply hj350-strassen.md]: I grown up in Strassen and since 13 years now in Mamer @Treverer

**Pull request merged**